keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/persistence/KeelReadOnlyRepository.kt

```diff
@@ -49,7 +50,7 @@ interface KeelReadOnlyRepository {
 
   fun isRegistered(name: String, type: ArtifactType): Boolean
 
-  fun artifactVersions(artifact: DeliveryArtifact, limit: Int = 30): List<String>
+  fun artifactVersions(artifact: DeliveryArtifact, limit: Int = DEFAULT_MAX_ARTIFACT_VERSIONS): List<String>
```

Comment:
This changes the default from 30 to 50, as discussed.

Reply:
where was this discussed?

Reply:
In Slack.

Reply:
@gal-yardeni I have a distinct memory of this being discussed somewhere and someone (maybe @erikmunson? @lorin?) suggesting it could be 50, but I can't find anything in Slack. If you feel strongly it should be 30, I'll change it back. Let me know, please.

Reply:
I tried to find this conversation but couldn't 😄 My personal opinion is that 30 versions are enough, that's the reason I was trying to find justifications for increasing it..